app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -20,8 +20,8 @@ youtube_video:
     id: mWuHMtyFrFU
     title: Subject knowledge enhancement explained
 navigation: 30.25
-navigation_title: Subject knowledge enhancement
-navigation_description: Find out how to improve your subject knowledge with a subject knowledge enhancement (SKE) course.
+navigation_title: Subject knowledge enhancement (SKE)
```

Comment:
This change is part of the page titles PR (https://github.com/DFE-Digital/get-into-teaching-app/pull/3715) but is fine to go out on this one: